tests/fixtures/civic-assertion-therapeutic-response-inline.yaml

```diff
@@ -0,0 +1,1121 @@
+    extensions:
+      - name: aliases
+        value:
+          - (2e)-N-(4-(3-Chloro-4-Fluoroanilino)-7-(((3s)-Oxolan-3-yl)Oxy)Quinoxazolin-6-yl)-4-(Dimethylamino)But-2-Enamide
+          - BIBW 2992
+          - BIBW-2992
+          - BIBW2992
+    id: civic.tid:146
+    mappings:
+      - coding:
+          code: C66940
+          id: ncit:C66940
+          system: https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=
+        relation: exactMatch
+    name: Afatinib
+  predicate: predictsSensitivityTo
+  subjectVariant:
+    aliases:
+      - LEU858ARG
+      - L813R
+      - LEU813ARG
+    description:
+      EGFR L858R has long been recognized as a functionally significant
+      mutation in cancer, and is one of the most prevalent single mutations in lung
+      cancer. Best described in non-small cell lung cancer (NSCLC), the mutation
+      seems to confer sensitivity to first and second generation TKI's like gefitinib
+      and neratinib. NSCLC patients with this mutation treated with TKI's show increased
+      overall and progression-free survival, as compared to chemotherapy alone.
+      Third generation TKI's are currently in clinical trials that specifically
+      focus on mutant forms of EGFR, a few of which have shown efficacy in treating
+      patients that failed to respond to earlier generation TKI therapies.
+    extensions:
+      - name: CIViC Molecular Profile Score
+        value: 379.0
+      - name: expressions
+        value:
+          - syntax: hgvs.g
+            value: NC_000007.13:g.55259515T>G
+          - syntax: hgvs.c
+            value: NM_005228.4:c.2573T>G
+          - syntax: hgvs.c
+            value: ENST00000275493.2:c.2573T>G
+          - syntax: hgvs.p
+            value: NP_005219.2:p.Leu858Arg
+      - name: CIViC representative coordinate
+        value:
+          chromosome: "7"
+          ensembl_version: 75
+          reference_bases: T
+          reference_build: GRCh37
+          representative_transcript: ENST00000275493.2
+          start: 55259515
+          stop: 55259515
+          type: coordinates
+          variant_bases: G
+    id: civic.mpid:33
+    mappings:
+      - coding:
+          code: rs121434568
+          system: https://www.ncbi.nlm.nih.gov/snp/
+        relation: relatedMatch
+    name: EGFR L858R
+    type: CategoricalVariant
+  type: VariantTherapeuticResponseProposition
+specifiedBy:
+  id: civic.method:2019
+  name: CIViC Curation SOP (2019)
+  reportedIn:
+    id: pmid:31779674
+    type: Document
+    name: Danos et al., 2019, Genome Med.
+    title:
+      Standard operating procedure for curation and clinical interpretation of
+      variants in cancer
+    doi: 10.1186/s13073-019-0687-x
+    pmid: "31779674"
+    aliases:
+      - CIViC curation SOP
+    urls:
+      - https://doi.org/10.1186/s13073-019-0687-x
+      - https://pubmed.ncbi.nlm.nih.gov/31779674/
+  type: Method
+  methodType: curation
```

Comment:
Took this from existing test fixture. Changed `methodType` to `curation` for GKS ClinVar This